migrate-from-vitess.md

```diff
@@ -0,0 +1,7 @@
+# Vitess to TiDB migration
+
+As the Vitess backend is based on MySQL you can use the regular MySQL migration tools like Dumpling, Lightning and DM which then need to be setup for each shard.
+
+In addition to this there is a [Debezium connector for Vitess](https://debezium.io/documentation/reference/connectors/vitess.html) which allows you to use Kafka Connect or Apache Flink to stream changes from Vitess to TiDB.
+
+As both Vitess and TiDB support the MySQL protocol and SQL dialect the amount of application level changes is expected to be small, except for things that directly manage sharding or other implementation specific things. To bridge the gap even more TiDB has support for the `VITESS_HASH()` function.
```

Comment:
I think this guide needs more work - did you want to maybe split it out into a different PR?

To migrate from Vitess you can either use the combined stream of servers that Vitess provides, or you can migrate from the individual MySQL servers that back Vitess and essentially ignore that it is Vitess (DM has the ability to merge manual shards).

I think if I were to attempt it I'd prefer to target the individual MySQL servers that back Vitess, but it would be great to have some discussion about it in the migration guide.

Reply:
https://github.com/pingcap/docs/pull/5783